src/components/AddPlaidBankAccount.js

```diff
@@ -100,6 +108,12 @@ class AddPlaidBankAccount extends React.Component {
     }
 
     componentDidMount() {
+        // If we're coming from Plaid OAuth flow then we need to reuse the existing plaidLinkToken
+        // Otherwise, clear the existing token and fetch a new one
+        if (this.props.receivedRedirectURI && this.props.plaidLinkOAuthToken) {
```

Comment:
Ran into an issue with this code while testing the `bank-account/personal` flow. Basically, what happens when we get redirected via OAuth back to NewDot `this.props.plaidLinkOAuthToken` is an empty string. Any ideas why?